src/components/WishlistCard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { useCart } from '../context/CartContext';
+import { useWishlist } from '../context/WishContext';
+
+function WishlistCard({_id,src,categoryName,price,originalPrice,title,discount,rating}){
```

Comment:
set item in wishlistcart 
after function start 
do like this 
const  {_id,src,categoryName,price,originalPrice,title,discount,rating} = item ;

Reply:
sure will do like this.